daemon/src/maker_cfd.rs

```diff
@@ -275,19 +284,22 @@ impl Actor {
             }
         };
 
-        // Update order in db
-        let order_id = cfd.order.id;
+        let (sender, receiver) = mpsc::unbounded();
+
+        if let SetupState::Active { .. } = self.setup_state {
```

Comment:
I think this early exit should be at the top of the function, we should potentially not even print the `Maker accepts an order` if we can't.